boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0_defconfig

```diff
@@ -7,6 +7,7 @@
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
+CONFIG_MFD=y
```

Comment:
Why is `CONFIG_MFD` being enabled by default here? Haven't we usually been selecting it from the LPI2C/LPUART/LPSPI drivers? Or is this required since LPUART and LPI2C can be enabled on the same LPFLEXCOMM?

Reply:
I have addressed this by enabling it in the `soc` `Kconfig` whenever `LPFlexcomm` is enabled.